.github/workflows/qa.yaml

```diff
@@ -8,6 +8,13 @@ on:
       - "release/*"
       - "renovate/*"
   pull_request:
+    paths-ignore:
```

Comment:
Right now this means we don't lint docs unless there are code changes, so that needs to change.